tests/post/steps/conftest.py

```diff
@@ -46,11 +46,17 @@ def _wait_for_status():
     )
 
 # }}}
+
+
+_PARSE_PODS_WITH_LABEL_STATUS = parsers.parse(
+    "pods with label '{label}' are '{expected_status}'")
+
+
 # Given {{{
 
 
-@given(parsers.parse("pods with label '{label}' are '{expected_status}'"))
-def check_pod_status(request, host, k8s_client, label, expected_status):
+@given(_PARSE_PODS_WITH_LABEL_STATUS)
+def given_check_pod_status(request, host, k8s_client, label, expected_status):
     ssh_config = request.config.getoption('--ssh-config')
```

Comment:
I know it's not related to your changes, but there's a `ssh_config` fixture to simplify this a bit:

```suggestion
def given_check_pod_status(ssh_config, host, k8s_client, label, expected_status):
```

Reply:
#2387